.github/workflows/goreleaser.yml

```diff
@@ -16,53 +16,56 @@ env:
 jobs:
   goreleaser:
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      packages: write
     steps:
-    - name: Checkout
-      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
-      with:
-        fetch-depth: 0
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+        with:
+          fetch-depth: 0
 
-    - uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5
-      with:
-        go-version: 1.24
-         
-    - name: Delete non-semver tags
-      run: 'git tag -d $(git tag -l | grep -v "^v")'
+      - uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5
+        with:
+          go-version: 1.24
 
-    - name: Set LDFLAGS
-      run: echo LDFLAGS="$(make ldflags)" >> $GITHUB_ENV
+      - name: Delete non-semver tags
+        run: 'git tag -d $(git tag -l | grep -v "^v")'
 
-    - name: Install syft binary executable
-      uses: anchore/sbom-action/download-syft@9246b90769f852b3a8921f330c59e0b3f439d6e9
+      - name: Set LDFLAGS
+        run: echo LDFLAGS="$(make ldflags)" >> $GITHUB_ENV
 
-    - name: Run GoReleaser on tag
-      uses: goreleaser/goreleaser-action@9c156ee8a17a598857849441385a2041ef570552
-      with:
-        distribution: goreleaser
-        version: latest
-        args: release --timeout 60m --clean
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        USER: ${{ github.actor }}
-        EMAIL: ${{ github.actor}}@users.noreply.github.com
+      - name: Install syft binary executable
+        uses: anchore/sbom-action/download-syft@e11c554f704a0b820cbf8c51673f6945e0731532
```

Comment:
This undoes a recent change. Please don't. I recommend rebasing your branch onto `main`, and looking out for this undo sneaking in. All would probably be easier without the whitespace changes.